cmake/cmake_test/add_dir.cmake

```
@@ -168,5 +176,10 @@ function(ct_add_dir _ad_test_dir)
                 -B "${_ad_test_dest_full_path}"
                 ${ADD_DIR_CMAKE_OPTIONS}
         )
+        if(ADD_DIR_LABEL AND NOT ADD_DIR_LABEL_NAME)
+            set_property(TEST ${_ad_test_name} PROPERTY LABELS ${_ad_test_dir})
+        elseif(ADD_DIR_LABEL_NAME)
+            set_property(TEST ${_ad_test_name} PROPERTY LABELS ${ADD_DIR_LABEL_NAME})
```

```suggestion
            set_property(TEST ${_ad_test_name} PROPERTY LABELS "${ADD_DIR_LABEL_NAME}")
```
I think you quotes in case users pass strings with spaces.